official-antistasi-community/A3-Antistasi
# Artillery marker improvements

## What type of PR is this.
1. [X] Bug
2. [ ] Change
3. [X] Enhancement

### What have you changed and why?
Information:
Added markers and hints to the friendly mortar strikes
Increased the accuracy of the marker lifetime for enemy artillery supports

closes #3245 

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Need to make sure every friendly marker is at the right scope and can be seen by everyone
Need to make sure the enemy arty markers last for an appropriate time in other modsets. Currently working in vanilla
********************************************************
Notes:
